File: MusicPlayer.v2/app/src/main/java/com/example/musicplayerv2/ThemeActivity.java
package com.example.musicplayerv2;

import android.content.SharedPreferences;
import android.preference.Preference;
import android.preference.PreferenceActivity;
import android.os.Bundle;
import android.preference.PreferenceFragment;
import android.preference.PreferenceManager;

public class ThemeActivity extends PreferenceActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: MusicPlayer.v2Kotlin/app/src/main/java/com/example/musicplayerv2/ThemeActivity.kt
package com.example.musicplayerv2

import android.content.SharedPreferences
import android.preference.Preference
import android.preference.PreferenceActivity
import android.os.Bundle
import android.preference.PreferenceFragment
import android.preference.PreferenceManager

class ThemeActivity : PreferenceActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferences)
    }
}
